xml_converter/integration_tests/src/testcase_loader.py

```diff
@@ -106,6 +106,19 @@ def load_testcase(path: str) -> Optional[Testcase]:
         print(f"Invalid Test, expecting string value for 'expected_returncode' in {path}")
         return None
 
+    expected_output_xml_path: str = os.path.join(path, "output_xml")
+    expected_output_proto_path: str = os.path.join(path, "output_proto")
+
+    if "exclude_output" in data:
+        for pack_type in data["exclude_output"]:
+            if pack_type == "xml":
+                expected_output_xml_path = ""
```

Comment:
This should be a unique value like `None` instead of treating `""` empty string as some special case path variable.